src/ImGui.NET/ImGuiNative.cs

```diff
@@ -272,7 +272,7 @@ public static unsafe class ImGuiNative
         public static extern void igTextWrapped(string fmt);
 
         [DllImport(cimguiLib)]
-        public static extern void igTextUnformatted(byte* text, byte* text_end);
+        public static extern void igTextUnformatted(string text, string text_end);
```

Comment:
I don't believe this is correct; these should be pointers to the beginning and end of the data, although the second parameter is allowed to be null. There's not really a way to pass the second parameter correctly if it is typed as a string. I think we should keep the PInvoke declaration the same, and in the `ImGui.TextUnformatted` method above, just pin the string and pass its address as the first parameter.